g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/schedule/SourceRequestCron.java

```diff
@@ -80,9 +81,17 @@ public void initSendFile() {
 
 		if (file.exists()) {
 			recoverDirectory(file);
+			logger.info("begin sync local recover directory component Model to remote");
```

Comment:
is it better to log the number of records?

Reply:
done